src/Date.php

```diff
@@ -29,7 +29,7 @@
  * - Uses either strptime() or DateTime to create a date from a format, depending on the format
  * - Uses strftime formatting for dates www.php.net/manual/en/function.strftime.php
  */
-class Date implements DateTimeInterface
```

Comment:
I agree, but since `DateTimeInterface` is now part of php 5.5, i get this error: `DateTimeInterface can't be implemented by user classes`. If we remove this, we could clean the fallback `DateTimeInterface` up.

What made me take this decision: http://php.net/manual/en/class.datetimeinterface.php#116219